x/concentrated-liquidity/swapstrategy/zero_for_one_test.go

```diff
@@ -121,7 +121,7 @@ func (suite *StrategyTestSuite) TestComputeSwapStepOutGivenIn_ZeroForOne() {
 			amountZeroInConsumed: defaultAmountZero.Ceil(),
 			// liquidity * (sqrtPriceNext - sqrtPriceCurrent)
 			expectedAmountOneOut:            defaultAmountOne,
-			expectedSpreadRewardChargeTotal: defaultAmountZero.Ceil().Quo(oneMinusDefaultSpreadFactor).Mul(defaultSpreadReward),
+			expectedSpreadRewardChargeTotal: defaultAmountZero.Ceil().Mul(defaultSpreadReward.Quo(oneMinusDefaultSpreadFactor)),
```

Comment:
Just making sure, these two should be the same value right

Reply:
They weren't due to rounding differences :sweat_smile: